pkgs/servers/eris-go/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, buildGoModule, fetchFromGitea }:
+
+buildGoModule rec {
+  pname = "eris-go";
+  version = "20230111";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "eris";
+    repo = pname;
+    rev = version;
+    hash = "sha256-lWEdGop90Ld3WvhEXEN0y0CEblsGzD7NfmiYmtkODOk=";
+  };
+
+  vendorHash = "sha256-DDV7LUnGnf24qQ2I9I4MDUx87s1+yDhisVz/Jw4XU6k=";
+
+  postInstall = "cp $out/bin/eris-get $out/bin/eris-put";
+  # eris-get is a multicall binary
+
+  checkPhase = "go test";
+
+  meta = src.meta // {
```

Comment:
Why do this?

```suggestion
  meta = {
```

Reply:
This implictly sets `meta.homepage` from `src`.

Reply:
Some people don't like this but I do, because it normalizes the definition of `homepage` and if stuff like `changelog` could be derived from the fetcher function then that would land in `meta` for free.